src/ResourceMover/tools/Resources/.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
The tools folder is not required to be checked in

Reply:
Removed the tools folder.